styles/components/registerV2.module.css

```diff
@@ -39,6 +39,13 @@
   white-space: nowrap;
 }
 
+.divider {
+  background-color: var(--background-light300, #F8F7F7);
```

Comment:
Only keep the variable, not both variable and color it's redudant. Variables should be defined in globals.css